src/main/webapp/dev/js/common/questionMcq.js

```diff
@@ -7,9 +7,12 @@ function addMcqOption(questionNum) {
 
     const curNumberOfChoiceCreated =
             parseInt($(`#${ParamsNames.FEEDBACK_QUESTION_NUMBEROFCHOICECREATED}-${questionNum}`).val(), 10);
+    const $choiceColumn = $(`#mcqChoices-${questionNum}`);
+    const $weightColumn = $(`#mcqWeights-${questionNum}`);
 
-    $(`
-    <div class="margin-bottom-7px" id="mcqOptionRow-${curNumberOfChoiceCreated}-${questionNum}">
+    const choiceFragment = (`
+    <div class="margin-bottom-7px mcqChoiceRow-${curNumberOfChoiceCreated}-${questionNum}"
```

Comment:
You forgot this class name and in the weight fragment below :P

Reply:
Oops! Fixing it.